activerecord/test/models/author.rb

```diff
@@ -149,7 +149,7 @@ def ratings
   has_many :categorized_posts, through: :categorizations, source: :post
   has_many :unique_categorized_posts, -> { distinct }, through: :categorizations, source: :post
 
-  has_many :nothings, through: :kateggorisatons, class_name: "Category"
+  has_many :nothings, through: :kategorisation, class_name: "Category"
```

Comment:
Why changed the nothing through association name? df97ed5e4c986be875248ec1b27d7acfc200ae9c

Reply:
I've added an explanation in another comment: https://github.com/rails/rails/pull/42333#discussion_r661369613